resources/views/livewire/home/trending-hashtags.blade.php

```diff
@@ -0,0 +1,5 @@
+<div>
+    @foreach ($hashtags as $hashtag)
+        <a class="bg-slate-600/20 p-4 shadow-2xl backdrop-blur font-medium me-2 py-1 px-2 rounded" href="{{ route('hashtag.show', $hashtag->name) }}">#{{ $hashtag->name }}</a>
```

Comment:
I believe this will need light/dark mode handling now.

Reply:
Yes @steven-fox Will add that too.